src/index.tsx

```diff
@@ -52,7 +60,17 @@ ReactDOM.render(
     // Globally disable ripples
     ripple={false}
   >
-    <ThemeProvider options={{ background, primary, secondary }}>
+    <ThemeProvider
+      options={{
+        background,
+        primary,
+        secondary,
+        error,
+        success,
```

Comment:
Please move `success`, `note`, `caution` out of the `--mdc-theme-` namespace and move them back to `--fire-color-`. `error` can stay since it overwrites material theme color, but `success` etc. won't play nicely with the rest of the ecosystem, for example `<Theme use="success">` won't pass type checking, and won't work even with `as any` since there is no underlying class for it. Similarly, nobody provides `successBg`, `onSuccess` etc., unlike `error`.

If applying `success` etc. to text ever become a common thing, we can roll out own helpers. Let's just stick to `var(--fire-color-success)` in CSS until then.

Reply:
sounds good :)

Reply:
Done